packages/react-composites/src/composites/CallWithChatComposite/Strings.tsx

```diff
@@ -17,6 +17,14 @@ export interface CallWithChatCompositeStrings {
    * {@link CallWithChatComposite} control bar People button label when checked
    */
   selectedPeopleButtonLabel: string;
+  /**
+   * {@link CallWithChatComposite} aria label for control bar People button when unchecked
+   */
+  peopleButtonTooltipOpenAriaLabel: string;
+  /**
+   * {@link CallWithChatComposite} aria label for control bar People button when checked
+   */
+  peopleButtonTooltipCloseAriaLabel: string;
```

Comment:
Not an issue for this PR, but I just noticed a lot of CommonControlBar strings are in the CallWithChatCompositeStrings object even though we also use them in CallComposite

Reply:
I noticed this as well, we will need to do a refactoring/repointing of strings in the future